website/docs/docs/modeling.mdx

```diff
@@ -198,8 +199,24 @@ For more information on how to use ordering and filtering in GraphQL, see [Conne
 ### Ordering
 
 All nodes can be ordered by id.
-Additionally, nodes of a specific type can also be ordered by all properties annotated with `@OrderProperty`. As those might not be unique, the `id` is used to create a strict total order.
+Additionally, nodes of a specific type can also be ordered by all properties annotated with `@OrderProperty`.
+As those might not be unique, the `id` is used to create a strict total order.
 By default, all data types which are [comparable in Cypher](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#cypher-ordering) are supported.
+Furthermore, `NodeProperty<*>` backed properties allow ordering by all orderable properties of the related node type, excluding `NodeProperty<*>` backed properties.
+
+:::caution
+
+Ordering on `NodeProperty<*>` does **NOT** take the authorization system into account.
+By making such a property orderable, it is possible to leak information about non-visible nodes.
```

Comment:
consider startup time check?

Reply:
I did
however, the permission which should be checked is only available at runtime (in theory, a user is allowed to provide a different permission each time)
a runtime check would be possible however
still, I'm not sure if this is desired as this would also disallow use cases which could be acceptable
Most importantly, this is only a partial information leak - only order properties can be leaked, and there may be use cases where you allow leaking one property to add some order functionality